code/game/machinery/doors/firedoor.dm

```diff
@@ -21,6 +21,7 @@
 	layer = BELOW_DOOR_LAYER
 	open_layer = BELOW_DOOR_LAYER
 	closed_layer = ABOVE_WINDOW_LAYER
+	movable_flags = MOVABLE_FLAG_Z_INTERACT
```

Comment:
this flag is for allowing things to be activated across zlevels? Why put it on firedoors?

Reply:
> this flag is for allowing things to be activated across zlevels? Why put it on firedoors?

![image](https://user-images.githubusercontent.com/41364830/119960848-13109280-bfae-11eb-878b-a27137b7c3c8.png)

Reply:
right its there, but why?